internal/hud/webview/convert.go

```diff
@@ -414,12 +407,14 @@ func protoPopulateResourceInfoView(mt *store.ManifestTarget, r *proto_webview.Re
 
 func runtimeStatus(res ResourceInfoView) RuntimeStatus {
 	_, isLocal := res.(LocalResourceInfo)
-	if isLocal {
+	_, isLocalPtr := res.(*LocalResourceInfo)
```

Comment:
this seems like it's working around a bug that should be fixed upstream. i don't see a legitimate reason to accept both types here